lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -1007,6 +1007,24 @@ def jsonrpc_settings_set(self, key, value):
             setattr(c, key, cleaned)
         return {key: cleaned}
 
+    def jsonrpc_settings_clear(self, key):
+        """
+        Clear daemon settings
+
+        Usage:
+            settings_clear (<key>)
+
+        Options:
+            None
+
+        Returns:
+            (dict) Updated dictionary of daemon settings
+        """
+        with self.conf.update_config() as c:
+            setattr(c, key, NOT_SET)
+            setting = self.conf.settings_dict[key]
+        return {key: setting}
```

Comment:
The get setting part doesn't need to be done inside the `update_config()` context manager, you can just inline it into the `return`:
```
return {key: self.conf.settings_dict[key]}
```